mobius-core/src/main/java/com/spotify/mobius/EffectRouterBuilder.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * -\-\-
+ * Mobius
+ * --
+ * Copyright (c) 2017-2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+package com.spotify.mobius;
+
+import com.spotify.mobius.functions.Consumer;
+
+/** TODO: document! */
+public interface EffectRouterBuilder<F, E> {
```

Comment:
I'm guessing you'll change the method names in here since we agreed overloading isn't the best way to go forward?

Reply:
Yes, done.